packages/web-frontend/src/screens/desktop/RootScreen/MainPage.js

```diff
@@ -50,8 +50,21 @@ const MapContainer = styled.div`
 `;
 
 const MainPage = ({ enlargedDialogIsVisible, isLoading, sidePanelWidth }) => {
-  const { isCustomLandingPage } = useCustomLandingPages();
-  if (isCustomLandingPage) return <LandingPage />;
+  const { isCustomLandingPageLoading, isCustomLandingPage } = useCustomLandingPages();
+
+  // Once the custom landing pages are loaded, hide the tupaia app loader in index.html
+  useEffect(() => {
+    if (isCustomLandingPageLoading === false) {
+      const el = document.getElementById('tupaia-spinner');
```

Comment:
the loading element is outside of the react app so we can't use a ref like we normally would